lib/cmd.js

```diff
@@ -0,0 +1,41 @@
+var commander = require('commander');
+var extract = require('./extract');
+var fs = require('fs');
+var packageJson = require('../package.json');
+
+commander
+.version(packageJson.version)
+.usage('[options] <file ...>')
+.option('-f, --function <regex>', 'Function pattern')
+.option('-k, --key <string>', 'Key')
+.parse(process.argv);
+
+var options = {
+    fun: new RegExp("^" + (commander['function'] || 'i18n') + "$"),
+    key: commander.key || '',
+    init: {},
+    warnings: []
+};
+
+for (var i = 0; i$ < commander.args.length; ++i$) {
```

Comment:
So...  I did this conversion back in October.  I have no idea why these are here.  They look wrong, don't they?  :P

Reply:
Aha.  So, apparently the way I did this was just to run the ls compiler, because it generated this line:

    for (i$ = 0, len$ = (ref$ = commander.args).length; i$ < len$; ++i$) {

Which I then fixed up, but not very well.  :)